const.ts

```diff
@@ -4,5 +4,5 @@
  * @create date 2018-08-26 02:37:53
  * @modify date 2018-08-26 03:26:37
  * @desc this projects constants
-*/
-exports.APP_NAME = 'Amazon Music'
\ No newline at end of file
+ */
+export default "Amazon Music" as string;
```

Comment:
Maybe keep the named export, rather than turning it into a default export. Just for future use

Reply:
--> 7a811ff

Reply:
--> 59d8c62